ci/cirrus.sh

```diff
@@ -21,22 +21,27 @@ valgrind --version || true
     --with-valgrind="$WITH_VALGRIND" \
     --host="$HOST" $EXTRAFLAGS
 
+make -j2
+
+# Print information about binaries so that we can see that the architecture is correct
+file *tests || true
```

Comment:
Why the "|| true" ?

Reply:
Otherwise the statement will fail if the wildcard does not match anything, and the files may not exist in every job. This is a little bit more meaningful in the line below when checking for benchmarks (because we have jobs with benchmarks disabled). But I just thought it's more consistent to apply the `|| true` in every line. In the future we may have special jobs without tests enabled (cttimetests only etc.)